SEED-platform/seed
# Timeseries Accumulative Updates

#### Any background context you want to provide?

#### What's this PR do?
- Tested for AWS deployment
- Optimize Portfolio Manager energy file and energy template upload process
- Delete energy data once the building is deleted
- Fix bug of duplicated monthly energy data in PostgreSQL
- Fix bug of invalid day of month
- Detect existence of KairosDB, skip KairosDB related operations if it is not found
- Merge with SEED/develop till PR #888 
#### How should this be manually tested?
- Deploy on AWS
- Add energy data of a building, then delete that building, the energy data should be also deleted
- Upload energy template with monthly data twice, the energy tab doesn't list duplicated data
- Stop KairosDB server, finer timeseries data cannot be assessed/inserted/deleted, but related page should work with no error
#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### Definition of Done:
